src/app/components/connect-addon-modal/connect-addon-modal.html

```diff
@@ -0,0 +1,82 @@
+
+      <form sf-schema="$ctrl.schema" sf-form="$ctrl.form" sf-model="$ctrl.model" ng-submit="$ctrl.submit(setupForm)" name="setupForm" ng-if="!$ctrl.isFormLoading"></form>
+      <mno-loader ng-if="$ctrl.isFormLoading"></mno-loader>
+
+    </div>
+  </div>
+
+  <!-- Chose Entities -->
+  <div ng-show="$ctrl.hasLinked && !$ctrl.hasChosenEntities">
+    <ul class="breadcrumb">
+      <li><a href="#">{{"mno_enterprise.templates.components.addon_connect.link_account.title" | translate}}</a></li>
+      <li>{{"mno_enterprise.templates.components.addon_connect.entities.select" | translate}}</li>
+      <li><a href="#" ng-click="$ctrl.forceSelectEntities()">{{"mno_enterprise.templates.components.addon_connect.sync.start_sync" | translate}}</a></li>
+    </ul>
+
+    <div class="modal-body">
+      <h4 class="text-center">{{"mno_enterprise.templates.components.addon_connect.entities.chose_entities" | translate}}</h4>
+
+      <table class="table-striped table-condensed">
+        <tr>
+          <th>{{"mno_enterprise.templates.components.addon_connect.entities.entity" | translate}}&nbsp</th>
+          <th>{{"mno_enterprise.templates.components.addon_connect.entities.to_app" | translate:{ appname: $ctrl.app.app_name} }}&nbsp</th>
+          <th>{{"mno_enterprise.templates.components.addon_connect.entities.to_connec" | translate:{ appname: $ctrl.app.app_name} }}&nbsp</th>
+        </tr>
+        <tr ng-repeat="entity in $ctrl.app.addon_organization.displayable_synchronized_entities">
+          <td>{{entity.connec_name}}&nbsp</td>
+          <td align="center"><input type="checkbox" ng-model="entity.can_push_to_external" checked="entity.can_push_to_external"/></td>
+          <td align="center"><input type="checkbox" ng-model="entity.can_push_to_connec" checked="entity.can_push_to_connec"/></td>
+        </tr>
+      </table>
+    </div>
+  </div>
+
+  <!-- Chose Entities -->
+  <div ng-show="$ctrl.hasLinked && $ctrl.hasChosenEntities">
+    <ul class="breadcrumb">
+      <li><a href="#">{{"mno_enterprise.templates.components.addon_connect.link_account.title" | translate}}</a></li>
+      <li><a href="#" ng-click="$ctrl.unselectEntities()">{{"mno_enterprise.templates.components.addon_connect.entities.select" | translate}}</a></li>
+      <li>{{"mno_enterprise.templates.components.addon_connect.sync.start_sync" | translate}}</li>
+    </ul>
+
+    <div class="modal-body">
+      <h4 class="text-center">Chose wether to synchronize historical data</h4>
+
+      <div>
+        <p translate="mno_enterprise.templates.components.addon_connect.sync.warning_hisorical_html" translate-values="{appname: $ctrl.app.name}"></p>
+        <input ng-model="$ctrl.historicalData" type="checkbox" display=inline-block></input>
+        <small ng-show="!$ctrl.historicalData"
+               translate="mno_enterprise.templates.components.addon_connect.sync.historical_unchecked" translate-values="{date: ($ctrl.date | amDateFormat:'dddd, MMMM Do YYYY, h:mm:ss a')}">
```

Comment:
For the date format, please use a format supporting user's locale.

Cf.http://momentjs.com/
<img width="780" alt="screen shot 2017-11-06 at 2 54 45 pm" src="https://user-images.githubusercontent.com/3941996/32424731-737ea2da-c302-11e7-9561-359e28c94568.png">